unattended_installer/install_functions/filebeat.sh

```diff
@@ -8,15 +8,15 @@
 
 function filebeat_configure(){
 
-    eval "curl -so /etc/filebeat/wazuh-template.json ${filebeat_wazuh_template} --max-time 300 ${debug}"
+    eval "common_curl -so /etc/filebeat/wazuh-template.json ${filebeat_wazuh_template} --max-time 300 --retry 5 --retry-delay 5 --fail ${debug}"
     if [ ! -f "/etc/filebeat/wazuh-template.json" ]; then
         common_logger -e "Error downloading wazuh-template.json file."
         installCommon_rollBack
         exit 1
     fi
-    
+
     eval "chmod go+r /etc/filebeat/wazuh-template.json ${debug}"
-    eval "curl -s ${filebeat_wazuh_module} --max-time 300 | tar -xvz -C /usr/share/filebeat/module ${debug}"
+    eval "common_curl -s ${filebeat_wazuh_module} --max-time 300 --retry 5 --retry-delay 5 --fail | tar -xvz -C /usr/share/filebeat/module ${debug}"
```

Comment:
Is there any place where flags like `--max-time 300` are different? If are always the same, we could use it statically in the function instead of using it in the command. Simplify the command view, and add the flags in the function. And review if other flags could apply it.

Reply:
Yes, in the Indexer component, there is an `--max-time 120`. The other parameters, `--retry`, `--retry-delay`, and `--fail`  differ from several `curl` calls.
Maybe we should specify all the arguments in the function call, in order to personalize it and change the values when needed.